src/lib/tradingView/utils.ts

```diff
@@ -29,7 +31,7 @@ const getOhlcValues = ({
 }) => {
   const useOrderbookCandles =
     orderbookCandlesToggleOn &&
-    trades === 0 &&
+    trades <= MIN_NUM_TRADES_FOR_ORDERBOOK_PRICES &&
```

Comment:
maybe rename to numTrades or tradesCount since trades kinda makes me think it's a list.

Reply:
ah yeah it's curr trades everywhere so will [follow uppp](https://linear.app/dydx/issue/CT-1138/rename-trades-numtrades) - will get ur unit test in there @yogurtandjam